tests/fs.rs

```diff
@@ -812,8 +812,8 @@ fn copy_file_returns_metadata_len() {
     let in_path = "in.txt";
     let out_path = "out.txt";
     check!(check!(tmp.create(&in_path)).write(b"lettuce"));
-    #[cfg(windows)]
-    check!(check!(tmp.create(tmp.join("in.txt:bunny"))).write(b"carrot"));
+    // #[cfg(windows)]
+    // check!(check!(tmp.create(tmp.join("in.txt:bunny"))).write(b"carrot"));
```

Comment:
Could you add a comment mentioning why this is commented out, with a TODO if appropriate?